src/floating-focus.js

```diff
@@ -83,11 +86,14 @@ export default class FloatingFocus {
 	enableFloatingFocus() {
 		this.container.classList.add('floating-focus-enabled');
 		this.floater.classList.add('enabled');
+		clearInterval(this.monitorElementPositionInterval);
+		this.monitorElementPositionInterval = setInterval(this.monitorElementPosition, 250);
```

Comment:
Maybe also make ```250``` a constant variable? I know it is not used multiple times, but it gives a clear overview of some of the time settings of this class. Just a suggestion, it's also fine like this.

Reply:
Yeah I can agree with this, it seems like an overall improvement to put all these in one place 👍